.github/workflows/test.yml

```diff
@@ -347,6 +352,42 @@ jobs:
       if: always()
       run: docker kill ${{steps.substrate.outputs.id}}
 
+  substrate-subxt:
+    name: Substrate Integration test with subxt
+    runs-on: ubuntu-20.04
+    needs: linux-x86-64
+    steps:
+    - name: Checkout sources
+      uses: actions/checkout@v2
+      # We can't run substrate as a github actions service, since it requires
+      # command line arguments. See https://github.com/actions/runner/pull/1152
+    - name: Start substrate
+      run: echo id=$(docker run -d -p 9944:9944 ghcr.io/hyperledger/solang-substrate-ci:e41a9c0 substrate-contracts-node --dev --ws-external) >> $GITHUB_OUTPUT
+      id: substrate
+    - uses: actions/download-artifact@master
+      with:
+        name: solang-linux-x86-64
+        path: bin
+    - run: |
+        chmod 755 ./bin/solang
+        echo "$(pwd)/bin" >> $GITHUB_PATH
+    - name: Install latest rust toolchain
+      uses: actions-rs/toolchain@v1
+      with:
+        toolchain: stable
+        default: true
+        override: true   
+    - run: solang compile --target substrate ../substrate/*.sol -o ./contracts/
+      working-directory: ./integration/subxt-tests
+    - run: solang compile --target substrate ../substrate/test/*.sol -o ./contracts/
+      working-directory: ./integration/subxt-tests
+    - name: Deploy and test contracts
+      run: cargo test -- --test-threads=1
```

Comment:
Why is `--test-threads=1` needed?

Reply:
I think I hit some node error not specifying this.
At least for my testing rig, it's way too fast for cargo to submit tx calls and the runtime is dropping it.

Reply:
Ah yes makes sense. Also we might hit issues with the CI nodes instant finality if we parallelize too much . I you are right we want to have this in.